### DISCUSSION TITLE
Reverse application usage

### DISCUSSION THREAD

**Discussion by devopg** [imzbf/md-editor-v3#709] [Ideas]
Is it possible to add the ability to use the library in both directions. That is, not only to edit markdown, but also to edit the final markup. For example, in the screenshot, I select the desired text in the final markup window and click bold.
?

![image](https://github.com/user-attachments/assets/f0588e1f-7bd6-411d-9429-4ef38e110fa9)

**Comment by imzbf** [comment]
https://github.com/imzbf/md-editor-v3/issues/316

This feature has been discussed before, and there are currently no plans.
